### PR TITLE
feat(CUST-CPC-409): Implementing buttons for UI

### PR DESCRIPTION
**Pull request by CrxJr**
JIRA: https://champlainsaintlambert.atlassian.net/jira/software/c/projects/CPC/boards/14?modal=detail&selectedIssue=CPC-409&sprint=77
## Context:
This change is been implemented so we can start adding the HTTP request required for Owner and Pet CRUD
## Changes
These changes only applied to the owner and pet forms.
The buttonsg required for the HTTP request have been added and just need to be configured
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
image
## Dev notes (Optional)
- The HTTP Request will be done after this pull request is implemented
- At the moment of creation docker-compose build does not work so the before changes cannot be showed
- Don't mind the changes to pet-form template
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

**Review by christopherl04 — APPROVED** [review]
Nice job cleaning up old comments that were polluting the code. I see you replaced a few lines of code to make it work, nice job!

File: api-gateway/src/main/resources/static/scripts/pet-form/pet-form.template.html [hunk 1]
```diff
@@ -40,4 +40,4 @@
             </button>
         </div>
     </div>
-</form>
+</form>x
```
Comment (Anh-Toine) [inline api-gateway/src/main/resources/static/scripts/pet-form/pet-form.template.html:43]:
```suggestion
You might want to remove the extra "x" on line 43.
```

Comment (Anh-Toine) [inline api-gateway/src/main/resources/static/scripts/pet-form/pet-form.template.html]:
```suggestion
You might want to remove the extra "x" on line 43.
```

**Review by Anh-Toine — DISMISSED** [review]
Great work cleaning up the UI code as well as adding new buttons to the UI that will enhance the user's experience when using the web app.

**Review by Louispoire — APPROVED** [review]
So far so good! Continue the great work. Hope we can implement the UI before the end of the week!

**Pull request merged**